samples/nrf_desktop/src/modules/CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ target_sources_ifdef(CONFIG_DESKTOP_BLE_ADVERTISING_ENABLE
 target_sources_ifdef(CONFIG_DESKTOP_BLE_SCANNING_ENABLE
 		     app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/ble_scan.c)
 
+target_sources_ifdef(CONFIG_SETTINGS
```

Comment:
This modules is required if settings are enabled, since it performs the settings load for whole application. We could move the load to a separate module e.g. settings.c, but that would complicate the initialization. What do you think?

Reply:
you are right, let's leave it as is